simapp/ante.go

```diff
@@ -33,13 +33,17 @@ func NewAnteHandler(options HandlerOptions) (sdk.AnteHandler, error) {
 		return nil, errors.New("sign mode handler is required for ante builder")
 	}
 
+	if options.Environment.RouterService == nil {
+		return nil, errors.New("router service is required for ante builder")
+	}
+
 	anteDecorators := []sdk.AnteDecorator{
-		ante.NewSetUpContextDecorator(), // outermost AnteDecorator. SetUpContext must be called first
+		ante.NewSetUpContextDecorator(options.Environment), // outermost AnteDecorator. SetUpContext must be called first
 		circuitante.NewCircuitBreakerDecorator(options.CircuitKeeper),
 		ante.NewExtensionOptionsDecorator(options.ExtensionOptionChecker),
-		ante.NewValidateBasicDecorator(options.AccountKeeper.GetEnvironment()),
```

Comment:
You think that is best not to expose an environment from a module? If so let's remove that getter then, I only added it to be less breaking.